recipes/instrain/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "instrain" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d6801888ea5de66c16423719d6fbfb98381b288676b18054ff4ae46f833f08ac
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```

Comment:
I think you can just leave `pip` and `python` in the `host` section, so please try that.